### PR TITLE
allow we see the alpha option output the bundle validation in another format

### PR DESCRIPTION
**Pull request by camilamacedo86**
closes: https://github.com/operator-framework/operator-sdk/issues/4562

### CODE REVIEW

File: internal/cmd/operator-sdk/bundle/validate/validate.go [hunk 1]
```diff
@@ -81,11 +81,6 @@ func (c *bundleValidateCmd) addToFlagSet(fs *pflag.FlagSet) {
 
 	fs.StringVarP(&c.outputFormat, "output", "o", internal.Text,
```
Comment (estroz) [inline internal/cmd/operator-sdk/bundle/validate/validate.go:82]: Can you add some help text saying something like
```
Note: output format types containing "alphaX" are subject to change and not covered by guarantees of stable APIs.
```

**Review by estroz — APPROVED** [review]
/lgtm

**Comment by estroz** [comment]
@camilamacedo86 could use a changelog `addition` type entry too.

**Comment by joelanford** [comment]
Has anyone actually used the existing alpha JSON format yet? If not we should we consider getting some folks testing with it before we unhide the flag?

**Comment by estroz** [comment]
@joelanford IMO because it's an alpha JSON spec, we should expose it then iterate based on feedback. Otherwise no one will know it exists (the current situation). In fact iirc it was intended for publication a few versions after it was added (v0.18) which never happened.

**Comment by camilamacedo86** [comment]
Hi @joelanford, 

For we use that we need to expose it as alpha. Otherwise, nobody has the chance to know that exists to test and check it. It will be used / required for a proposal that I have been working on.  

@estroz @jmrodri can we get it merged?

File: internal/cmd/operator-sdk/bundle/validate/validate.go (hunk 1 above)
Comment (camilamacedo86) [reply internal/cmd/operator-sdk/bundle/validate/validate.go:82]: Done

**Review by jmrodri — APPROVED** [review]
/lgtm

**Pull request merged**